app/Http/Controllers/Api/V1/CurrikiGo/ExtractXAPIJSONController.php

```diff
@@ -26,7 +27,7 @@ class ExtractXAPIJSONController extends Controller
      * @param  LRSStatementsSummaryDataRepositoryInterface  $lrsStatementsSummaryDataRepositoryInterface
      * @return void
      */
-    public function runJob(ActivityRepositoryInterface $activityRepository, LRSStatementsDataRepositoryInterface $lrsStatementsRepository, LRSStatementsSummaryDataRepositoryInterface $lrsStatementsSummaryDataRepositoryInterface)
+    public function runJob(ActivityRepositoryInterface $activityRepository, LRSStatementsDataRepositoryInterface $lrsStatementsRepository, LRSStatementsSummaryDataRepositoryInterface $lrsStatementsSummaryDataRepositoryInterface, GoogleClassroomRepositoryInterface $googleClassroom)
```

Comment:
Doc blocks need to be updated.
Also params be on new line to avoid horizontal scroll.

Reply:
Fixed